### ISSUE TITLE
in __init__ self.files = [PackagePath(i) for i in self.raw["files"]] KeyError: 'files'

### ISSUE THREAD

**Issue by kkaskie** [brentvollebregt/auto-py-to-exe#439]
Hello, 

I keep running into this error "line 97, in __init__ self.files = [PackagePath(i) for i in self.raw["files"]] KeyError: 'files'". The interface will load but none of the buttons are operational. 

**Describe the bug**
A clear and concise description of what the bug is. If you have an error message, please paste it here (not in a screenshot).
Traceback (most recent call last):

Initially ran auto-py-to-exe-2.9.0 and things worked fine. Some time has past sence last used and the below error occured. Updated the packages and running auto-py-to-exe-2.41.0 now with the same outcome. 

  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\eel\__init__.py", line 281, in _process_message
    return_val = _exposed_functions[message['name']](*message['args'])
  File "C:\Users\cwb725\OneDrive - MT\Desktop\auto-py-to-exe-2.9.0\auto_py_to_exe\ui.py", line 45, in initialise
    'options': packaging.get_pyinstaller_options(),
  File "C:\Users\cwb725\OneDrive - MT\Desktop\auto-py-to-exe-2.9.0\auto_py_to_exe\packaging.py", line 43, in get_pyinstaller_options
    parser = __get_pyinstaller_argument_parser()
  File "C:\Users\cwb725\OneDrive - MT\Desktop\auto-py-to-exe-2.9.0\auto_py_to_exe\packaging.py", line 22, in __get_pyinstaller_argument_parser
    from PyInstaller.building.build_main import __add_options as add_build_options
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\building\build_main.py", line 28, in <module>
    from PyInstaller.building.api import COLLECT, EXE, MERGE, PYZ
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\building\api.py", line 32, in <module>
    from PyInstaller.building.splash import Splash  # argument type validation in EXE
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\building\splash.py", line 24, in <module>
    from PyInstaller.utils.hooks import tcl_tk as tcltk_utils
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\utils\hooks\__init__.py", line 1272, in <module>
    from PyInstaller.utils.hooks import conda as conda_support  # noqa: F401
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\utils\hooks\conda.py", line 385, in <module>
    distributions = _init_distributions()
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\utils\hooks\conda.py", line 380, in _init_distributions
    dist = Distribution(path)
  File "C:\Users\cwb725\AppData\Roaming\Python\Python39\site-packages\PyInstaller\utils\hooks\conda.py", line 97, in __init__
    self.files = [PackagePath(i) for i in self.raw["files"]]
KeyError: 'files'

**To Reproduce**
Steps to reproduce the behavior (install dependencies, do this in the auto-py-to-exe UI):
1. Execute run.py

**Your Environment:**
- Python version (`python --version`): 3.19.16
- auto-py-to-exe version (`pip show auto-py-to-exe`): 2.9.0 and 2.41.0
- PyInstaller version (`pip show pyinstaller`): 6.1.0
- Eel version (`pip show eel`): 0.14.0

**Comment by brentvollebregt** [comment]
Line 97 in what file? That doesn't look like it comes from this project. 

Please fill out the template you cleared so we have all the information required to look into this.

**Comment by kkaskie** [comment]
> Line 97 in which file? That doesn't look like it comes from this project.
> 
> Please fill out the template you cleared so we have all the information required to look into this.

Initial comment has been updated.

**Comment by brentvollebregt** [comment]
Those reproduction steps don't give me anything to work off, unfortunately; all I can recommend is to:

1. Create a new virtual environment
2. Install your dependencies in this virtual environment
3. Verify your script runs in this virtual environment
4. Install auto-py-to-exe in this virtual environment
5. Run auto-py-to-exe from this virtual environment and package your application
    - Make sure it's from this virtual environment and not the one you're currently using
    - You should be able to use `python -m auto_py_to_exe` after you have activated the virtual environment

This should hopefully remedy any version errors and stop PyInstaller from trying to bundle conda as it might be getting in the way.

_Also just a heads up, the latest version of Python is 3.12 😄_

**Comment by nikbog-mec** [comment]
I've started having this issue as well. When I run the run.py, I get the KeyError: 'files' as well. I've successfully ran this in the past so I'm not sure what would've changed. I tried to run this on a different machine with a fresh download of auto-py-to-exe and I get the same issue. Currently running on Python 3.9.16. Not sure if this has any web-based dependencies - If it does, it could possibly be a firewall issue in the company.
